packages/ui/components/index-page/AddButton.tsx

```diff
@@ -0,0 +1,15 @@
+import { PlusOutlined } from '@ant-design/icons';
+import { ADD_TASK } from 'constants/labels';
+
+const AddButton = () => {
```

Comment:
Can we use Ant Button?

Reply:
we will remove antd, so will change accordingly :)